src/core_atmosphere/physics/mpas_atmphys_driver_convection.F

```diff
@@ -172,6 +173,38 @@ subroutine allocate_convection(configs)
        if(.not.allocated(kbot_shallow_p) ) allocate(kbot_shallow_p(ims:ime,jms:jme)     )
        if(.not.allocated(ktop_deep_p)    ) allocate(ktop_deep_p(ims:ime,jms:jme)        )
 
+       if(.not.allocated(dx_p)           ) allocate(dx_p(ims:ime,jms:jme)               )
```

Comment:
are these all just copies of those is GF? if so, why not just add GFL to the case statement above? and only have a GFL section for additional variables needed by GFL?